<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>propertiesPanel</class>
 <widget class="QWidget" name="propertiesPanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>350</width>
    <height>750</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Properties Panel</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="ButtonLayout">
     <property name="spacing">
      <number>2</number>
     </property>
     <item>
      <widget class="QPushButton" name="Accept">
       <property name="text">
        <string>&amp;Apply</string>
       </property>
       <property name="icon">
        <iconset resource="../pqComponents.qrc">
         <normaloff>:/pqWidgets/Icons/pqUpdate16.png</normaloff>:/pqWidgets/Icons/pqUpdate16.png</iconset>
       </property>
       <property name="shortcut">
        <string>Alt+A</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Reset">
       <property name="text">
        <string>&amp;Reset</string>
       </property>
       <property name="icon">
        <iconset resource="../pqComponents.qrc">
         <normaloff>:/pqWidgets/Icons/pqCancel16.png</normaloff>:/pqWidgets/Icons/pqCancel16.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Delete">
       <property name="text">
        <string>&amp;Delete</string>
       </property>
       <property name="icon">
        <iconset resource="../../../Widgets/Resources/QtWidgets.qrc">
         <normaloff>:/QtWidgets/Icons/pqDelete16.png</normaloff>:/QtWidgets/Icons/pqDelete16.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Help">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../pqComponents.qrc">
         <normaloff>:/pqWidgets/Icons/pqHelp16.png</normaloff>:/pqWidgets/Icons/pqHelp16.png</iconset>
       </property>
       <property name="shortcut">
        <string>Ctrl+H</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="SearchBoxLayout">
     <property name="spacing">
      <number>2</number>
     </property>
     <item>
      <widget class="pqSearchBox" name="SearchBox" native="true">
       <property name="settingKey" stdset="0">
        <string notr="true">showAdvancedProperties</string>
       </property>
       <property name="advancedSearchEnabled" stdset="0">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>332</width>
        <height>689</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="ScrollAreaLayout" stretch="0,0,0,0,0,0,1">
       <property name="margin">
        <number>0</number>
       </property>
       <item>
        <widget class="QWidget" name="PropertiesButtons" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout">
          <property name="spacing">
           <number>6</number>
          </property>
          <property name="margin">
           <number>0</number>
          </property>
          <item>
           <widget class="pqExpanderButton" name="PropertiesButton">
            <property name="text" stdset="0">
             <string>Properties</string>
            </property>
            <property name="checked" stdset="0">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="PropertiesCopy">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="maximumSize">
             <size>
              <width>50</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="toolTip">
             <string>Copy properties</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="../pqComponents.qrc">
              <normaloff>:/pqWidgets/Icons/pqCopy22.png</normaloff>:/pqWidgets/Icons/pqCopy22.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="PropertiesPaste">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="maximumSize">
             <size>
              <width>50</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="toolTip">
             <string>Paste copied properties</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="../pqComponents.qrc">
              <normaloff>:/pqWidgets/Icons/pqPaste22.png</normaloff>:/pqWidgets/Icons/pqPaste22.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="PropertiesRestoreDefaults">
            <property name="maximumSize">
             <size>
              <width>50</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="toolTip">
             <string>Restore application default setting values</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="PropertiesSaveAsDefaults">
            <property name="maximumSize">
             <size>
              <width>50</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="toolTip">
             <string>Save current settings values as default</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="PropertiesFrame">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Plain</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="DisplayButtons" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <property name="spacing">
           <number>6</number>
          </property>
          <property name="margin">
           <number>0</number>
          </property>
          <item>
           <widget class="pqExpanderButton" name="DisplayButton">
            <property name="text" stdset="0">
             <string>Display</string>
            </property>
            <property name="checked" stdset="0">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="DisplayCopy">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="maximumSize">
             <size>
              <width>50</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="toolTip">
             <string>Copy display properties</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="../pqComponents.qrc">
              <normaloff>:/pqWidgets/Icons/pqCopy22.png</normaloff>:/pqWidgets/Icons/pqCopy22.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="DisplayPaste">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="maximumSize">
             <size>
              <width>50</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="toolTip">
             <string>Paste copied display properties</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="../pqComponents.qrc">
              <normaloff>:/pqWidgets/Icons/pqPaste22.png</normaloff>:/pqWidgets/Icons/pqPaste22.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="DisplayRestoreDefaults">
            <property name="maximumSize">
             <size>
              <width>50</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="toolTip">
             <string>Restore application default setting values for display</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="DisplaySaveAsDefaults">
            <property name="maximumSize">
             <size>
              <width>50</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="toolTip">
             <string>Save current display settings values as default</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="DisplayFrame">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Plain</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="ViewButtons" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <property name="spacing">
           <number>6</number>
          </property>
          <property name="margin">
           <number>0</number>
          </property>
          <item>
           <widget class="pqExpanderButton" name="ViewButton">
            <property name="text" stdset="0">
             <string>View</string>
            </property>
            <property name="checked" stdset="0">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="ViewCopy">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="maximumSize">
             <size>
              <width>50</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="toolTip">
             <string>Copy view properties</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="../pqComponents.qrc">
              <normaloff>:/pqWidgets/Icons/pqCopy22.png</normaloff>:/pqWidgets/Icons/pqCopy22.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="ViewPaste">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="maximumSize">
             <size>
              <width>50</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="toolTip">
             <string>Paste view properties</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="../pqComponents.qrc">
              <normaloff>:/pqWidgets/Icons/pqPaste22.png</normaloff>:/pqWidgets/Icons/pqPaste22.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="ViewRestoreDefaults">
            <property name="maximumSize">
             <size>
              <width>50</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="toolTip">
             <string>Restore application default setting values for view</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="ViewSaveAsDefaults">
            <property name="maximumSize">
             <size>
              <width>50</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="toolTip">
             <string>Save current view settings values as default</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="ViewFrame">
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Plain</enum>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>0</width>
           <height>9</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>pqSearchBox</class>
   <extends>QWidget</extends>
   <header>pqSearchBox.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>pqExpanderButton</class>
   <extends>QFrame</extends>
   <header location="global">pqExpanderButton.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../pqComponents.qrc"/>
  <include location="../../../Widgets/Resources/QtWidgets.qrc"/>
 </resources>
 <connections/>
</ui>
